packages/tools/src/tools/annotation/CircleROITool.ts

```diff
@@ -984,9 +984,21 @@ class CircleROITool extends AnnotationTool {
           modalityUnitOptions
         );
 
+        const invXRadiusSq =
+          ellipseObj.xRadius !== 0 ? 1 / ellipseObj.xRadius ** 2 : 0;
```

Comment:
Why bother with the test here - just get the square of the radius, which is 0 if the radius is 0.  Seems like extra work to handle that case.